.travis.yml

```diff
@@ -35,6 +35,11 @@ jobs:
       script: deploy/index.js militaryaerospace aerilon
       install: skip
 
+      - stage: deploy
```

Comment:
Need to fix indentation here to match the other lines, TravisCI build failed due to the parse error

Reply:
Forgot to reply that I had pushed up this fix